### DISCUSSION TITLE
Errors for C++ tests in Android studio

### DISCUSSION THREAD

**Discussion by cyber-toad** [organicmaps/organicmaps#5138] [Q&A] [answered]
I set up project in Android studio, but have an issue with C++ tests: they don't see any includes so are full of errors.
I tried "Sync now" option, reimport/rebuild of project but it doesn't change anything.
![image](https://github.com/organicmaps/organicmaps/assets/128428520/b9837426-e296-4a27-ad08-a315b8e0aefd)

For prod (non-test) c++ code notification is displayed, but there are no errors.
![image](https://github.com/organicmaps/organicmaps/assets/128428520/8c9fe947-45aa-4e03-961e-ec980bd863ca)

Could anyone advise how to fix it please?
Thanks.

**Comment by biodranik** [comment, marked as answer]
We are not using Android Studio for C++ development yet and didn't set it up/tune it up for C++ development. You may try to discover it yourself and share your findings.

We are using Qt Creator, Visual Studio, or XCode at the moment.

**Reply by cyber-toad** [reply]
Thanks, it is clear now. I was able to open project in CLion IDE.
